src/main/java/edu/pw/eiti/pik/elasticsearch/UserESRepository.java

```diff
@@ -0,0 +1,22 @@
+package edu.pw.eiti.pik.elasticsearch;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.elasticsearch.annotations.Query;
+import org.springframework.data.elasticsearch.repository.ElasticsearchRepository;
+
+import edu.pw.eiti.pik.user.User;
+
+public interface UserESRepository extends ElasticsearchRepository<User, Long>{
```

Comment:
thos should be inside user package because it's user-related repository

Reply:
It should not be in user package because Spring will mismatch it for JPA repository and fail to generate query

Reply:
Why mismatch? It has different name and extends different interface. But if the package is really that important, maybe user/elasticsearch will be good location?

Reply:
It would still require moving UserRepository to e.g. user/db or specifying repository classes instead of packages